include/secp256k1.h

```diff
@@ -584,7 +584,8 @@ SECP256K1_API SECP256K1_WARN_UNUSED_RESULT int secp256k1_ec_pubkey_create(
  *           secp256k1_ec_seckey_verify. 1 otherwise
  *  Args:   ctx:        pointer to a context object
  *  In/Out: seckey:     pointer to the 32-byte private key to be negated. The private
- *                      key should be valid according to secp256k1_ec_seckey_verify
+ *                      key should be valid according to secp256k1_ec_seckey_verify.
+ *                      Value becomes undefined if this function returns 0.
```

Comment:
Can we say "unspecified"? "Undefined" comes with that weird terrifying sound of undefined behavior.